_sources/governance.rst.txt

```diff
@@ -0,0 +1,169 @@
+PySAL project governance
+========================
+
+The purpose of this document is to formalize the governance process used by the PySAL project in both ordinary and extraordinary situations, and to clarify how decisions are made and how the various elements of our community interact, including the relationship between open source collaborative development and work that may be funded by for-profit or non-profit entities.
+
+
+The Project¶
+------------
+
+PySAL, the Python spatial analysis library, is an open source cross-platform project (The Project). The goal of The Project is to develop open source software for geospatial data science in Python, and in particular the PySAL library and its Sub-Packages. The Software developed by The Project is released under the BSD (or similar) open source license, developed openly and hosted on public GitHub repositories under the pysal GitHub organization.
+
+The Project is developed by a team of distributed developers, called Contributors. Contributors are individuals who have contributed code, documentation, designs or other work to The Project. Anyone can be a Contributor. Contributors can be affiliated with any legal entity or none. Contributors participate in the project by submitting, reviewing and discussing GitHub Pull Requests and Issues and participating in open and public Project discussions on GitHub, Gitter, and other channels. The foundation of Project participation is openness and transparency.
+
+The Project Community consists of all Contributors and Users of The Project. Contributors work on behalf of and are responsible to the larger Project Community and we strive to keep the barrier between Contributors and Users as low as possible.
```

Comment:
... all Contributors and users of the software developed by Contributors (hereinafter referred to as "Users") of The Project.